internal/command/deploy.go

```diff
@@ -16,8 +18,18 @@ func Deploy(
 	logger.Title("Find existing change set")
 
 	changeSet, err := client.FindExistingChangeSet(ctx, stack)
+
 	if err != nil {
```

Comment:
I'm a bit rusty, are there any scenarios where `err != nil` apart from the change set not existing, where we should continue to short circuit?

Reply:
Yeah, there's a few exceptional things that could be bubbled from AWS SDK calls. Would be the pattern here to define a custom error? My go is rustier (in the sense it never existed) than your stratus

Reply:
I'm too far detached to speak to idiomatic Go, but I'd be supportive of smuggling through an additional `changeSet == nil, err == nil` state.